packages/guides/resources/template/html/body/table.html.twig

```diff
@@ -18,10 +18,12 @@
         {% for tableRow in tableRows %}
             <tr>
                 {% for column in tableRow.columns %}
-                    <td{% if column.colSpan > 1 %} colspan="{{ column.colSpan }}"{% endif %}{% if column.rowSpan > 1 %} rowspan="{{ column.rowSpan }}"{% endif %}>
+                <td{% if column.colSpan > 1 %} colspan="{{ column.colSpan }}"{% endif %}{% if column.rowSpan > 1 %} rowspan="{{ column.rowSpan }}"{% endif %}>
+                    {%- if column.children is not empty %}
                         {%- for child in column.children -%}
-                            {{- renderNode(child) -}}
-                        {% endfor %}</td>
+                                {{- renderNode(child) -}}
+                        {% endfor %}
+                    {%- else %}&nbsp;{% endif %}</td>
```

Comment:
This can be simplified by the for else construction supported by twig:

```
    {%- for child in column.children -%}
        {{- renderNode(child) -}}
    {% else %}
         &nbsp;
    {% endfor %}
```
This is not a requirement to be changed before merging this, but just so you know this construct exists.

Reply:
Thanks for the tip. I've changed my code accordingly